core/src/main/java/brooklyn/catalog/CatalogLoadMode.java

```diff
@@ -20,8 +20,11 @@
 
 import org.slf4j.LoggerFactory;
 
+import brooklyn.catalog.internal.CatalogInitialization;
 import brooklyn.entity.rebind.persister.PersistMode;
 
+/** @deprecated since 0.7.0 replaced by {@link CatalogInitialization} */
```

Comment:
given how long it has been since the `0.6.0` release i tend not to remove things which have been in place for more than a few weeks.  (hopefully soon we *will* be releasing on that cadence...)